docs/updating/nim-migration/migration-to-new-installation-methods.md

```diff
@@ -0,0 +1,117 @@
+---
+category: nim-migration
+order: 10
+menu-title: Migrating to new installation methods
+meta-title: Migrating to new installation methods | CKEditor5 documentation
+meta-description: Learn how to upgrade your CKEditor5 project to the new installation methods.
+modified_at: 2024-06-06
+---
+
+# Migrating to new installation methods
+
+<info-box hint>
+This guide will help you migrate your CKEditor&nbsp;5 project to the new installation methods introduced in CKEditor&nbsp;5 version 42.0.0. If you created your project after the release of CKEditor&nbsp;5 v42.0.0, you likely already use the new installation methods and can skip this guide.
+</info-box>
+
+With the introduction of the new installation methods, we have greatly simplified the process of using CKEditor&nbsp;5 in your project by reducing the number of possible installation paths and eliminating most of the limitations that were present in the old methods.
+
+This guide will show you how to migrate your project from the old installation methods to one of the new ones. First, let's take a look at what the new editor installation looks like to better understand what has changed.
+
+## FAQ
+
+//TODO
+
+## New installation methods
+
+We have reduced the number of possible installation methods to just two: **npm packages and browser builds**. Unlike before, both methods no longer require you to add dozens of individual packages or JavaScript bundles to get the editor up and running. Instead, you can import the editor and all our open source plugins from `ckeditor5` and the premium features from `ckeditor5-premium-feature`.
```

Comment:
@filipsobol I miss more details in here:

*   Split of styles (potential performance improvements).
*   bundler-agnostic setup
*   etc.

So that we convince people here to migrate.